backend/controllers/bulkAppointmentsSlipsController.js

```diff
@@ -22,13 +22,23 @@ module.exports = ({ elite2Api, logError }) => async (req, res) => {
 
     const createdBy = forenameToInitial(userDetails.name)
     const offenderNumbers = prisonersListed.map(prisoner => prisoner.offenderNo)
-    const offenderSummaries = await elite2Api.getOffenderSummaries(res.locals, offenderNumbers)
+    const chunkedOffenderNumbers = chunkArray(offenderNumbers, 100)
+
+    const offenderSummaryApiCalls = chunkedOffenderNumbers.map(offendersChunk => ({
+      getOffenderSummaries: elite2Api.getOffenderSummaries,
+      offenders: offendersChunk,
+    }))
+
+    const offenderSummaries = await Promise.all(
```

Comment:
Would you be better of doing:
`(await Promise.all(stuff)).reduce(`

Reply:
That won't work.  Think it makes more sense for the Promise.all to be chained with `.then` which contains the responses so they can then be reduced them into one array.

Reply:
As discussed, didn't see you'd wrapped it in `()`